internal/gonut/cmd/push.go

```diff
@@ -131,6 +132,7 @@ func init() {
 
 	pushCmd.PersistentFlags().StringVarP(&deleteSetting, "delete", "d", "always", "Delete application after push: always, never, on-success")
 	pushCmd.PersistentFlags().StringVarP(&summarySetting, "summary", "s", "short", "Push summary detail level: quiet, short, full")
+	pushCmd.PersistentFlags().BoolVarP(&pingSetting, "ping", "p", false, "Ping application after push to check the responding statuscode")
```

Comment:
I would like to head @tareqmamari opinion on this one, too. My take is that I would opt for the ping test being the default and the flag gives the end user the possibility to skip the check.